lib/filters/api.js

```diff
@@ -34,17 +51,20 @@ const getFilter = (name) => {
     }
   }
 }
+
 const setEnvironment = (env) => {
   environment = env
-  while (filtersToAddToEnv.length > 0) {
-    const { name, fn, config } = filtersToAddToEnv.shift()
-    addFilterToEnvironment(name, fn, config)
+  while (whenEnvIsAvailable.length > 0) {
+    const fn = whenEnvIsAvailable.shift()
+    fn()
   }
 }
+
 module.exports = {
   external: {
     addFilter,
-    getFilter
+    getFilter,
+    addGlobal
```

Comment:
Let's add a note about this to the release notes

Reply:
Done